geoengine/workflow.py

```diff
@@ -285,14 +285,21 @@ def plot_chart(self, bbox: QueryRectangle) -> VegaLite:
 
         return VegaLite(vega_spec)
 
-    def __get_wcs_tiff_as_memory_file(self, bbox: QueryRectangle, timeout=3600) -> rasterio.io.MemoryFile:
+    def __get_wcs_tiff_as_memory_file(
+        self,
+        bbox: QueryRectangle,
+        timeout=3600,
+        force_no_data_value=None
```

Comment:
can you add a type annotation?

Reply:
[82fc21d](https://github.com/geo-engine/geoengine-python/pull/60/commits/82fc21da4cf428ac6c02945b0a1d010245431c27)